### PR TITLE
Follow up to CCR docs

### PR DESCRIPTION
**Pull request by BaurzhanSakhariev**
Follow up for [11821](https://github.com/crate/crate/pull/11821)

## Checklist

 - [x] Added an entry in `CHANGES.txt` for user facing changes
 - [x] Updated documentation & `sql_features` table for user facing changes
 - [x] Touched code is covered by tests
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

File: docs/admin/logical-replication.rst [hunk 1]
```diff
@@ -24,25 +24,27 @@ Logical replication is useful for the following use cases:
...
+
 - Ensure High Availability if one cluster becomes unavailable.
 
 - Replicating between different major versions of CrateDB via chain of
-  compatible versions. Adjacent major versions are compatible.
-
-- Consolidating multiple clusters into a single one for aggregated reports.
+  compatible versions. Adjacent major versions are compatible. Logical
+  replication is available from 4.7, thus one can replicate clusters with
+  versions 4.7 and higher.
```
[the remark this answers is not in the document]
Comment (BaurzhanSakhariev) [reply docs/admin/logical-replication.rst]: Hi @seut, could you please take a look to this follow up?

File: docs/admin/logical-replication.rst [hunk 2]
```diff
@@ -24,25 +24,24 @@ Logical replication is useful for the following use cases:
...
-and might also be described as a replication set.
+A publication is the upstream side of logical replication and it's created on
+the cluster which acts as a data source.
 
-Publications are different from schemas and do not affect how the table is
-accessed. Each table can be added to multiple publications if needed.
-Publications can only contain tables. All operation types
-(INSERT, UPDATE, DELETE) are replicated.
+Each table can be added to multiple publications if needed. Publications can
+only contain tables. All operation types (INSERT, UPDATE, DELETE) are
```
Comment (seut) [inline docs/admin/logical-replication.rst:43]:
```suggestion
only contain tables. All operation types (INSERT, UPDATE, DELETE) are
```
```suggestion
only contain tables. All operation types (INSERT, UPDATE, DELETE and schema changes) are
```

File: docs/admin/logical-replication.rst [hunk 3]
```diff
@@ -63,23 +62,26 @@ process on the subscriber cluster. The subscriber cluster behaves in the same
...
-is lost. This means that the data has to be resynchronized afterwards.
-
 Published tables must not exist on the subscriber. A cluster cannot subscribe
-to a table on another cluster if it exists already on its side. Only regular
-tables (including partitions) may be the target of replication. For example,
+to a table on another cluster if it exists already on its side, therefore it's
+not possible to drop and re-create a subscription without starting from scratch
+i.e removing all replicated tables.
+
+Only regular tables (including partitions) may be the target of replication. For example,
```
Comment (seut) [inline docs/admin/logical-replication.rst:84]: minor: Looks like this violates the 80 char line limit?

**Review by seut — COMMENTED** [review]
Looks good from my side, just added 2 suggestions.

**Review by seut — APPROVED** [review]
👍

**Comment by BaurzhanSakhariev** [comment]
retest this please

**Pull request merged**